src/UI/templates/default/Button/tpl.standard.html

```diff
@@ -0,0 +1 @@
+<button type="button" class="btn btn-default" href="{ACTION}" {DISABLED}>{LABEL}</button>
```

Comment:
Thx for the href.

A HTML validator in the tests would indeed be nice. If someone makes a pull request with an according method to ILIAS_UI_TestBase i will be happy to integrate it in the tests of the currently implemented components.